extras/upgrade_activities.sh

```diff
@@ -29,6 +29,9 @@ function my_exit()
   if [[ ${GBD_STARTED} -ne 0 ]]
   then
     pkill -15 gluster-blockd
+
+    # let's wait until the child process is finished 
```

Comment:
There is one white space right after this comment, in both the places.

Reply:
Will fix it.
Thanks.